apps/node-status/src/components/frontend-referral.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button } from '@penumbra-zone/ui';
+import { devFrontend, prodFrontend } from '../constants.ts';
+
+export const FrontendReferral = () => {
+  const onClickHandler = () => {
+    window.open(import.meta.env.MODE === 'production' ? prodFrontend : devFrontend);
```

Comment:
why not use `process.env.NODE_ENV`? Not available in Vite?

Reply:
Because this code will be executed in a browser environment, getting env variables that way will throw. Vite provides this as a kind analogous way to access env variables with a client side app.